.taskcluster.yml

```diff
@@ -158,15 +159,15 @@ tasks:
                                       - index.project.${project}.v2.branch.${head_branch}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
                                       - index.project.${project}.v2.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
                       scopes:
-                          $if: 'tasks_for == "github-push"'
-                          then:
-                              # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
-                              - 'assume:repo:${repoUrl[8:]}:branch:${short_head_branch}'
-                          else:
-                              $if: 'tasks_for == "github-pull-request"'
-                              then:
+                          $switch:
+                              'tasks_for == "github-push"':
+                                  # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
+                                  - 'assume:repo:${repoUrl[8:]}:branch:${short_head_branch}'
+                              'tasks_for == "github-issue-comment"':
+                                  - 'assume:repo:github.com/'
```

Comment:
This should assume the same thing as `github-pull-request` events. That way anything we grant to PRs we also grant to issue comments (I don't see any benefit to granting things to one but not the other).